README.md

```diff
@@ -53,8 +53,11 @@ Now add the card like this:
     - type: 'custom:spotify-card'
       client_id: <YOUR CLIENT ID>
       limit: <optional number of playlists to retrieve (default 10)>
+      device: <optional name of a device to pre-select>
 ```
 
+If you add the `device` setting, the card will select it by default and will not display the dropdown menu.
```

Comment:
Do you think it would make sense to have a small mini icon that would force the select menu to be shown in the case the player is not available?

Reply:
Yeah, could be, unless we can get the status from the api ?